### DISCUSSION TITLE
What's the most efficient way to store hex strings?

### DISCUSSION THREAD

**Discussion by gd87429** [duckdb/duckdb#8632] [Q&A]
Need to store hex strings in parquet files, optimizing for the smallest file size.
I'm guessing `blob` of bytes is better than an actual hex string, or is there something better? Any downsides?

**Comment by Tishj** [comment]
If they're hex then I imagine converting them to integers would be most efficient

**Reply by gd87429** [reply]
Thanks for the reply. I also considered that, but then from my understanding `HUGEINT` max value would not be able to support `u256`, right?
